config/routes.rb

```diff
@@ -1,12 +1,22 @@
 Rails.application.routes.draw do
-  root :to => "tasks#tasks", as: "all_tasks"
-  post "/", to: "tasks#tasks"
+
+  # root :to => "tasks#tasks", as: "all_tasks"
+  # post "/", to: "tasks#tasks"
+
+  get "tasks", to: "tasks#tasks", as: "all_tasks"
+  root :to => "users#new", as: "homepage"
+
   patch "/tasks/:id/stop" => "tasks#stop_stask", as: "stop_task"
   patch "/tasks/:id/done" => "tasks#done_stask", as: "done_task"
   patch "/tasks/:id/restart" => "tasks#restart_stask", as: "restart_task"
   patch "/tasks/:id/start" => "tasks#start_stask", as: "start_task"
```

Comment:
you can use nest resources
https://guides.rubyonrails.org/routing.html#adding-more-restful-actions